packages/react/src/views/AttachmentHandler/AudioAttachment.js

```diff
@@ -1,18 +1,135 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import { Box } from '@embeddedchat/ui-elements';
+import { css } from '@emotion/react';
+import { Box, Avatar, useTheme } from '@embeddedchat/ui-elements';
 import AttachmentMetadata from './AttachmentMetadata';
+import RCContext from '../../context/RCInstance';
 
-const AudioAttachment = ({ attachment, host, variantStyles }) => (
-  <Box>
-    <AttachmentMetadata
-      attachment={attachment}
-      url={host + (attachment.title_url || attachment.audio_url)}
-      variantStyles={variantStyles}
-    />
-    <audio src={host + attachment.audio_url} width="100%" controls />
-  </Box>
-);
+const AudioAttachment = ({
+  attachment,
+  host,
+  type,
+  authorIcon,
+  authorName,
+  variantStyles,
+}) => {
+  const { RCInstance } = useContext(RCContext);
+  const { colors } = useTheme();
+  const getUserAvatarUrl = (icon) => {
```

Comment:
prop name such as authorUrl here will be better.

Reply:
Okay will change this